libs/domain/push-notification/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@spryker-oryx/push-notification",
+  "version": "0.0.1",
+  "main": "./src/index.js",
+  "typings": "./src/index.d.ts",
+  "author": "Spryker Sytems GmbH",
+  "license": "See license in LICENSE.md",
+  "peerDependencies": {
+    "@spryker-oryx/core": "*",
+    "@spryker-oryx/di": "*",
+    "@spryker-oryx/router": "*",
+    "@spryker-oryx/ui": "*",
```

Comment:
I don't see dependencies to `router` and `ui` in this package so we should remove those.